deploy/hackathon-docker/docker-compose.yml

```diff
@@ -0,0 +1,25 @@
+version: '3'
+services:
+  web:
```

Comment:
If use docker-compose，we can deploy 2 service: server and client, and expose different port

Reply:
i thought about it and i chose the easier way, i suggest we merge this one as step 1, and i will separate them afterward.

Reply:
what's more, localhost is a problem to be taken care of if they are separated, so far, there are 6 in config_sample.py respectively, but i have no idea how many left.

Reply:
Always need to consider it even the two are in the same container because of Guacamole and publicly-accesible OHP server. The easist way is to use ENV variables. As I suggested, we need another config-docker.py to read ENV variables.

Reply:
Some like:
```
ADD ./open-hackathon-server /opt/open-hackathon/open-hackathon-server
cp /opt/open-hackathon/open-hackathon-server/config-docker.py /opt/open-hackathon/open-hackathon-server/config.py
```
And in this file, some variables are from ENV

Reply:
if using Pod in Kube, a container can access other using localhost in the same pod

but in Docker-Compose, a container access others using service name（need add links: https://docs.docker.com/compose/compose-file/#links）

BUT, We can't rely on “ localhost”， I suggest add a configurable item.